IHP/Controller/Context.hs

```diff
@@ -143,21 +143,23 @@ instance HasField "frameworkConfig" ControllerContext FrameworkConfig where
 --
 -- This can be useful to customize the log formatter for all actions of an app:
 --
--- > import IHP.Log.Types
+-- > import IHP.Log.Types as Log
 -- > import IHP.Controller.Context
 -- > 
 -- > instance InitControllerContext WebApplication where
 -- >     initContext = do
 -- >         let defaultLogger :: Logger = ?context.frameworkConfig.logger
--- >         let withUserIdLogger :: Logger = { Log.formatter = userIdFormatter defaultLogger.formatter }
+-- >         let withUserIdLogger = defaultLogger { Log.formatter = userIdFormatter defaultLogger.formatter } :: Logger
```

Comment:
The weird thing is that this results in an error

`let withUserIdLogger :: Logger = defaultLogger { Log.formatter = userIdFormatter defaultLogger.formatter }`

![image](https://github.com/digitallyinduced/ihp/assets/125707/a2511ee1-8f5e-4560-9738-fc0de3222889)

Reply:
Any clue why? ^^

Reply:
To clarify, the code in the PR is correct. But if moving the ` :: Logger` to be next to `let withUserIdLogger` it errors out.

Reply:
I cannot reproduce this error. Do you have a branch or an PR where I can try this out?